### PR TITLE
Words refactor

### PR DESCRIPTION
**Pull request by BKSteve**
Refactor a bunch of variables to snake_case

## Summary by CodeRabbit

- **New Features**
	- Enhanced robustness in error handling by ensuring `error_count` and `warning_count` variables are initialized correctly in the local scope.
- **Style**
	- Updated function naming conventions across the codebase to adhere to Python's PEP 8 guidelines, improving readability and consistency.
- **Bug Fixes**
	- Ensured that changes in naming conventions do not affect the functionality of the application, maintaining the logic for retrieving show names and search strings.

### CODE REVIEW

**Comment by BKSteve** [comment]
@coderabbitai review

File: sickchill/gui/slick/views/layouts/main.mako
Comment (BKSteve) [inline sickchill/gui/slick/views/layouts/main.mako:1]: @miigotu grateful for input on this. I have not run SC recently and threw an error when testing just now.
`UnboundLocalError: cannot access local variable 'error_count' where it is not associated with a value`

Comment (miigotu) [reply sickchill/gui/slick/views/layouts/main.mako:1]: strict_undefined needs set to False in the code initializing mako - UNDEFINED is a special variable for mako

Comment (BKSteve) [reply sickchill/gui/slick/views/layouts/main.mako:1]: I understand UNDEFINED is a special variable for mako, and it's set in `sickchill/views/common.py` 
```
strict_undefined=settings.DEVELOPER or settings.DEBUG,
```
which is true as in testing mode.
This is a pain as basically the GUI doesn't run in debug or developer mode.

Note: The merge of tpbUHD had this change too, so what's the right way?

**Comment by BKSteve** [comment]
@coderabbitai configuration

**Pull request merged**